libr/fs/p/fs_squash.c

```diff
@@ -1,7 +1,9 @@
 /* radare - LGPL - Copyright 2016 - pancake */
 
 #include <r_fs.h>
+#if !(__WINDOWS__ && !defined(__CYGWIN__))
```

Comment:
this is not necessary because it include r_fs and this includes r_util, so just having this #if block in r_util is enough